### PR TITLE
docs: minutes for 2016-11-23 CTC meeting

### PR DESCRIPTION
**Pull request by joshgav**
cc @nodejs/ctc

### CODE REVIEW

File: meetings/2016-11-23.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+## Q/A on public channels
+
+### Q: What do you think about Yarn?
+
+@Fishrock123: Competition is good.
+
+@jasnell: Lots of promise, perf is phenomenal, will take a bit of time to
+mature.
+
+@mhdawson: Impressive, still some maturity to gain.
```
Comment (mhdawson) [inline meetings/2016-11-23.md]: make this:
@mhdawson: Its good to have options, it probably still needs some time to mature

File: meetings/2016-11-23.md [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+inheritance for now. It would require removing `Buffer` without new.
+
+It’s become obvious that we’ll never be able to deprecate `Buffer` without new.
+We may need a new API to support subclassability, it wouldn’t be too difficult.
+
+@trott: Perhaps the CTC decision-making process is not effective in dealing with
+this issue. Perhaps we should delegate to a small subset of CTC and non-CTC
+people who have strong opinions and interest and can close on a recommendation.
+
+@mhdawson: At least we have consensus to revert the deprecation.
```
Comment (mhdawson) [inline meetings/2016-11-23.md]: should be:
@mhdawson:  I don't think I've heard objections in our discussion today to reverting the deprecation.

**Review by mhdawson — APPROVED** [review]
LGTM with a couple of minor changes

**Review by Trott — APPROVED** [review]

File: meetings/2016-11-23.md [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+@Fishrock123: Compatibility is impressive.
+
+### Note from YouTube: Private repos in big companies cannot easily change and need to be considered.
+
+---
+
+
+## Upcoming Meetings
+
+* CTC: 2016-Nov-30, 2000 UTC (10am in Austin)
```
Comment (Trott) [inline meetings/2016-11-23.md]: This should be `(2pm in Austin)` and not 10am.

**Comment by joshgav** [comment]
Made corrections and landed in 4827dd1. Thanks!

**Pull request closed without merging**